src/compiler/crystal/loader.cr

```diff
@@ -89,7 +89,7 @@ class Crystal::Loader
 
     @search_paths.each do |directory|
```

Comment:
We could change this to `@search_paths.any?` but I didn't want to introduce a diff bigger than necessary here